src/main/docker/Dockerfile

```diff
@@ -14,26 +14,17 @@
 # limitations under the License.
 #
 
-FROM opensuse:latest
+FROM cafapi/opensuse-tini:1.0
 
 # Update the OS packages and install Java and cURL
 RUN zypper -n refresh && \
     zypper -n update && \
     zypper -n install java-1_8_0-openjdk curl
```

Comment:
I'd rather that we moved curl into the base tini image also.  We're going to want that for node.js and everything.

Reply:
cURL has been removed